vm-setup/roles/v1aX_integration_test/templates/controlplane_centos.yaml

```diff
@@ -204,6 +204,19 @@ spec:
         owner: root:root
         permissions: '0600'
         content: {{ SSH_PUB_KEY_CONTENT }}
+      - path: /tmp/install-kubernetes-binaries.sh
+        permissions: '0755'
+        content: |
+          #!/bin/bash 
+          RELEASE="${KUBERNETES_BINARIES_VERSION}"
+          # Get executables
+          curl -L --remote-name-all https://storage.googleapis.com/kubernetes-release/release/${RELEASE}/bin/linux/amd64/{kubeadm,kubelet,kubectl}
+          chmod +x {kubeadm,kubelet,kubectl}
+          # Move binaries to a folder in PATH
+          mv {kubeadm,kubelet,kubectl} /usr/local/bin
+          curl -sSL "https://raw.githubusercontent.com/kubernetes/kubernetes/${RELEASE}/build/debs/kubelet.service" | sed "s:/usr/bin:/opt/bin:g" > /etc/systemd/system/kubelet.service
```

Comment:
kubelet is not copied to /opt/bin, you copied it to /usr/local/bin. this issue is in all files

Reply:
It should not make a difference as long as the binaries are in the path. 
the remove /opt/bin on purpose.

Reply:
it does make a difference. if you open https://raw.githubusercontent.com/kubernetes/kubernetes/v1.17.3/build/debs/kubelet.service, you will get this content 
```
[Unit]
Description=kubelet: The Kubernetes Node Agent
Documentation=http://kubernetes.io/docs/

[Service]
ExecStart=/usr/bin/kubelet
Restart=always
StartLimitInterval=0
RestartSec=10

[Install]
WantedBy=multi-user.target
```
by running this sed command, you replace /usr/bin/kubelet by /opt/bin/kubelet. whether kubelet is in your path or not, systemd will look for /opt/bin/kubelet. that does not exist, hence the CI failure

Reply:
your sed command is a search and replace. replace `/usr/bin` by `/opt/bin`

Reply:
Good point. 
But, the CI should not reach that level yet. There were two other issues that would prevent it reaching it there. Working on those.